test/test_certdirectory.py

```diff
@@ -261,6 +261,14 @@ def test_list_for_product_match(self):
         res = self.d.list_for_product('123456789')
         self.assertTrue(isinstance(res, list))
 
+    def test_list_for_product_name(self):
+        res = self.d.list_for_product_name('123')
+        self.assertIsInstance(res, list)
+
+    def test_list_for_product_name_case(self):
```

Comment:
This unit test does not test case non-sensitivity at all. Please modify it.

Reply:
I took a stab at this, can you check to see if it works as expected?